src/main/java/sirius/biz/web/BizController.java

```diff
@@ -150,11 +152,30 @@ protected void load(WebContext ctx, Entity entity) {
      */
     protected void load(WebContext ctx, Entity entity, Column... properties) {
         Set<String> columnsSet = Arrays.stream(properties).map(Column::getName).collect(Collectors.toSet());
+        load(ctx, entity, columnsSet);
+    }
+
+    private void load(WebContext ctx, Entity entity, Set<String> properties) {
+        boolean hasError = false;
+
         for (Property property : entity.getDescriptor().getProperties()) {
-            if (columnsSet.contains(property.getName())) {
-                property.parseValue(entity, ctx.get(property.getName()));
+            String propertyName = property.getName();
+
+            if (properties.contains(propertyName)) {
+                try {
+                    property.parseValue(entity, ctx.get(propertyName));
+                } catch (IllegalArgumentException e) {
```

Comment:
sollte man die IllegalArgumentException nicht in parseValue in property catchen 
und hier eine HandledException - dann muss man net 2 catchen